docs/reference/release-notes/7.10.asciidoc

```diff
@@ -160,13 +160,13 @@ Also see <<breaking-changes-7.10,Breaking changes in 7.10>>.
 [[security-updates-7.10.0]]
 === Security updates
 
-* Elasticsearch versions before 7.10.0 and 6.8.14 have an information
-disclosure issue when audit logging and the emit_request_body option is
-enabled. The {es} audit log could contain sensitive information
+* {es} versions before 7.10.0 and 6.8.14 have an information
+disclosure issue when audit logging and the `emit_request_body` option are
+enabled. The {es} audit log could contain sensitive information,
 such as password hashes or authentication tokens. This could allow an
 {es} administrator to view these details.
 You must upgrade to {es} version 7.10.2 to obtain the fix.
```

Comment:
@jrodewig I missed this - this should be 7.10.0, yes?

Reply:
:doh: Sorry about that I overlooked this too. Yes. This should be 7.10.0.

Reply:
However, I'm not sure it's worth updating... It should be impossible for users to obtain an earlier patch version now.

Reply:
I pushed a fixed anyway.